components/form/SelectFormControl.js

```diff
@@ -6,18 +6,20 @@ function SelectFormControl({ id, label, options, value, onChange }) {
 
   return (
     <div className="form-control">
-      <label className="label" htmlFor={id}>
-        <span className="label-text">{label}</span>
-      </label>
+      {label && (
+        <label className="label" htmlFor={id}>
+          <span className="label-text">{label}</span>
+        </label>
+      )}
       <select
         className="select select-bordered w-full max-w-xs select-sm"
         id={id}
         name={id}
         onChange={handleChange}
         value={value}
       >
-        {options.map(({ id: optionId, label: optionLabel }) => (
-          <option key={optionId} value={optionId}>
+        {options.map(({ disabled, label: optionLabel, value: optionValue }) => (
```

Comment:
The auto-expanding nature of variables in modern JavaScript is such a nice feature. 😄